sdk/anomalydetector/ai-anomaly-detector/package.json

```diff
@@ -65,9 +65,9 @@
     "@azure/core-http": "^1.2.0",
     "@azure/core-auth": "^1.1.3",
     "@azure/logger": "^1.0.0",
-    "@opentelemetry/api": "^0.10.2",
+    "@opentelemetry/api": "^0.16.0",
     "tslib": "^2.0.0",
-    "@azure/core-tracing": "1.0.0-preview.9"
+    "@azure/core-tracing": "1.0.0-beta.1"
```

Comment:
I thought we had decided to leave old previews as "preview" until they GA'd?

Reply:
that is true

Reply:
Not an editorial choice on my part - I'm just matching what's in the [package.json](https://github.com/Azure/azure-sdk-for-js/blob/eeaa146191bb6d094e45251f0265e08dcd072f16/sdk/core/core-tracing/package.json#L3) in master.

Should I revert it to preview.10 instead?

Reply:
Strange. Yeah it should be preview.10 unless there's some compelling reason